.github/workflows/int_test.yml

```diff
@@ -36,27 +36,32 @@ on:
         required: false
         default: false
 
-env:
-  CLUSTER_NAME: "az-iot-ops-test-cluster-${{ github.run_id }}-${{ github.run_attempt }}"
-  KV_NAME: "opstestkv-${{ github.run_id }}-${{ github.run_attempt }}"
-  CLUSTER_RG: ${{ inputs.resource_group }}
-  CUSTOM_LOCATIONS_OID: "51dfe1e8-70c6-4de5-a08e-e18aff23d815"
-  EXTENSION_SOURCE_DIRECTORY: "./azure-iot-ops-cli-extension"
-  K3S_VERSION: "v1.28.5+k3s1"
-
 permissions:
   # required for OpenID federation
   contents: 'read'
   id-token: 'write'
 
 jobs:
   integration_test:
+    env:
+      CLUSTER_NAME: "az-iot-ops-test-cluster${{ github.run_id }}${{ matrix.feature }}"
+      # TODO - KV Name has a hard limit of 24 chars
+      KV_NAME: "opstestkv${{ github.run_id }}${{ matrix.feature }}"
+      CLUSTER_RG: ${{ inputs.resource_group }}
+      CUSTOM_LOCATIONS_OID: "51dfe1e8-70c6-4de5-a08e-e18aff23d815"
+      EXTENSION_SOURCE_DIRECTORY: "./azure-iot-ops-cli-extension"
+      K3S_VERSION: "v1.28.5+k3s1"
+    strategy:
+      matrix:
+        feature: [std, mq]
+        include:
+          - feature: mq
```

Comment:
nit mq looks too vanilla to me, can we identify this like "mq-ins", or something a little more specific to the variant that is being tested.

Reply:
Sure can, initially I tried to keep this as short as possible due to a length in the naming scheme I used for JIT key vault creation - but there are better ways to handle that :)